YAMP.Core/Operators/OpDefinitions.cs

```diff
@@ -0,0 +1,112 @@
+﻿namespace YAMP
+{
+    using System;
+
+    /// <summary>
+    /// This public class declares some exposable operator's symbols and levels
+    /// </summary>
+    public static class OpDefinitions
+    {
+        public const int ArgsOperatorLevel = 1000;
+        public const String ArgsOperator = "(";
+
+        public const int MemberOperatorLevel = 990; //Moved from 10000 to 990 to be under ArgsOperatorLevel
+        public const String MemberOperator = ".";
+
+        public const int FactorialOperatorLevel = 980; //Moved from 1000 to 980 to be under ArgsOperatorLevel (and allow space for MemberOperatorLevel)
+        public const String FactorialOperator = "!";
+
+        public const int PreIncOperatorLevel = 950; //Moved from 999 to 950 to give space below FactorialOperatorLevel and ArgsOperatorLevel
+        public const String PreIncOperator = "++";
+
+        public const int PostIncOperatorLevel = 950; //Moved from 999 to 950 to give space below FactorialOperatorLevel and ArgsOperatorLevel
+        public const String PostIncOperator = "++";
+
+        public const int PreDecOperatorLevel = 950; //Moved from 999 to 950 to give space below FactorialOperatorLevel and ArgsOperatorLevel
+        public const String PreDecOperator = "--";
+
+        public const int PostDecOperatorLevel = 950; //Moved from 999 to 950 to give space below FactorialOperatorLevel and ArgsOperatorLevel
+        public const String PostDecOperator = "--";
+
+        public const int InvOperatorLevel = 900; //Moved from 995 to 900 to give some space.
+        public const String InvOperator = "~";
+
+        public const int PowerOperatorLevel = 100;
+        public const String PowerOperator = "^";
+
+        public const int AdjungateOperatorLevel = 100;
+        public const String AdjungateOperator = "'";
+
+        public const int TransposeOperatorLevel = 100;
+        public const String TransposeOperator = ".'";
+
+        public const int ModuloOperatorLevel = 30;
+        public const String ModuloOperator = "%";
+
+        public const int LeftDivideOperatorLevel = 20;
+        public const String LeftDivideOperator = @"\";
+
+        public const int RightDivideOperatorLevel = 20;
+        public const String RightDivideOperator = "/";
+
+        public const int MultiplyOperatorLevel = 10;
+        public const String MultiplyOperator = "*";
+
+        public const int NegateOperatorLevel = 7;
+        public const String NegateOperator = "-";
+
+        public const int PosateOperatorLevel = 7;
+        public const String PosateOperator = "+";
+
+        public const int MinusOperatorLevel = 6; //Shouldn't be the same as PlusOperatorLevel ?
```

Comment:
Notice that I 've used the "+" and "-" for Union and Except.
In Sets theory we cannot change the operations order. It would produce different results.
For example:
with "+" <=> Union, and "-" as Except, 
1) (a + a) - a = {}, but
2) a + (a - a) = a
Since "-" has a higher level than "+", the expression is evaluated as in 2).
You can even see it with my implementation:
a=newset("A"); a + a - a
output is: "Name = (A+(A-A)), Sorted = False, Count = 0"

I would expect, without thinking, that the expression would evaluate from left to right, but as it seems, many authors suggest that parêntesis should always be used, so this was, at least, the operations are always done in the specified order...
Maybe I could have chosen the "/" operator to do the "Except" operation. In that case the "similarities" with the numeric operators levels would be better.
But it seems more "natural" to use the "-", but it's disputable.
Do you have an opinion on that?
Now is the right time to set it.